src/instructlab/model/serve.py

```diff
@@ -56,7 +56,12 @@
 @click.option(
     "--vllm-args",
     type=str,
-    help="Specific arguments to pass into vllm. The value passed into this flag must be surrounded by double quotes.\n Example: --vllm-args '--dtype auto --enable-lora'",
+    multiple=True,
+    help=(
+        "Specific arguments to pass into vllm. Each arg must be passed "
+        "separately and surrounded by quotes.\n"
+        " Example: --vllm-args='--dtype auto' --vllm-args='--enable-lora'"
```

Comment:
Isn't `'--dtype auto'` really 2 args?

```suggestion
        " Example: --vllm-args='--dtype' --vllm-args='auto' --vllm-args='--enable-lora'"
```

or should it be `'--dtype=auto'`?